### PR TITLE
220 improve client performance

### PR DESCRIPTION
**Pull request by wenwenchenbosch**
solve #220

### CODE REVIEW

File: kuksa_feeders/dbc2val/dbcfeeder.py [hunk 1]
```diff
@@ -44,6 +44,7 @@
 kuksa.start()
 kuksa.authorize()
 
+
```
Comment (SebastianSchildt) [inline kuksa_feeders/dbc2val/dbcfeeder.py:47]: Why changing this?

File: kuksa_viss_client/__init__.py [hunk 2]
```diff
@@ -189,5 +210,6 @@ async def mainLoop(self):
     # Thread function: Start the asyncio loop
     def run(self):
         self.loop = asyncio.new_event_loop()
+        self.loop.set_debug(True)
```
Comment (SebastianSchildt) [inline kuksa_viss_client/__init__.py:213]: Is this needed in production?

**Review by SebastianSchildt — COMMENTED** [review]
See small comments.

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
Much faster 👍   🥇 

Should merge, but I still feel we should not run in asyncio debug mode

File: kuksa_viss_client/__init__.py (hunk 2 above)
Comment (wenwenchenbosch) [reply kuksa_viss_client/__init__.py:213]:
```suggestion
```

**Review by SebastianSchildt — APPROVED** [review]

**Pull request merged**